cmd/vitconfig/vitconfig.go

```diff
@@ -1027,7 +1035,7 @@ func main() {
 		kit.FatalOn(err, "vcli.DbInit", kit.B2S(out))
 
 		// populate the database with already dumped data
-		out, err = vcli.CsvDataLoad(vitDb, fundsFile.Name(), proposalsFile.Name(), votePlansFile.Name())
+		out, err = vcli.CsvDataLoad(vitDb, fundsFile.Name(), proposalsFile.Name(), *challengesPath, votePlansFile.Name())
```

Comment:
As far as I see, there is no need to change anything from the csv itself, so we just need to pipe the path to the vit-servicing-station cli (fund_id is added to the challenges there).
Could you confirm @eugene-babichenko ?

Reply:
Yes, you're right.

Reply:
Shouldn't `err` be processed here or propagated to the caller?

Reply:
It is processed in the next line.

Reply:
Ah, this `kit.FatalOn` tricked me, I expected `if err == nil`.